app/controllers/application_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
```

Comment:
Another non-correction: keep in mind this line because it *will* fail in the future. As soon as you get an error about CSRF tokens during a request, google this.